Authentification/ProfessionalProfilePictureViewController.m

```diff
@@ -98,4 +101,11 @@ - (IBAction)didSignUp:(id)sender {
     [self performSegueWithIdentifier:@"secondSegue" sender:nil];
 }
 
+- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
+    if ([segue.identifier isEqual:@"sessionSegue"]){
+        [self performSegueWithIdentifier:@"sessionSegue" sender:nil];
```

Comment:
are we sure about this? `prepareForSegue` is only called when `performSegueWIth...` is triggered. So the perform segue should already been fired before getting to this method.

Reply:
You are right! This is left-over code.